Modified gameplay.java

```diff
@@ -0,0 +1,221 @@
+
+
+    public Gameplay() {
+        map = new WallGenerator(5, 8);
+        addKeyListener(this);
+        setFocusable(true);
+        setFocusTraversalKeysEnabled(false);
+        timer = new Timer(delay, this);
+        timer.start();
+
+    }
+
+    public void paint(Graphics g) {
+        // background
+        g.setColor(Color.black);
+        g.fillRect(1, 1, 692, 592);
+
+        //drawing map
+        map.draw((Graphics2D) g);
+        // border 
+        g.setColor(Color.yellow);
+        g.fillRect(0, 0, 3, 592);
+        g.fillRect(0, 0, 692, 3);
+        g.fillRect(680, 0, 3, 592);
+
+        //scores
+        g.setColor(Color.white);
+        g.setFont(new Font("serif", Font.BOLD, 25));
+        g.drawString("" + score + "/200", 590, 30);
+
+        //paddle
+        g.setColor(Color.green);
+        g.fillRect(playerX, 550, 100, 8);
+
+        //the ball
+        g.setColor(Color.yellow);
+        g.fillOval(ballposX, ballposY, 20, 20);
+
+        if (play == false && totalBricks == 40) {
+            //game start message
+            g.setColor(Color.white);
+            g.setFont(new Font("serif", Font.PLAIN, 20));
+            g.drawString("PRESS LEFT OR RIGHT TO START", 190, 350);
+
+            //game name
+            g.setColor(Color.white);
+            g.setFont(new Font("serif", Font.PLAIN, 40));
+            g.drawString("BREAKOUT BALL", 180, 240);
+
+            //hide ball
+            g.setColor(Color.black);
+            g.fillRect(110, 340, 40, 40);
+
+            //hide bricks
+            g.setColor(Color.black);
+            g.fillRect(80, 40, 540, 150);
+        }
+
+        if (totalBricks <= 0) {
+            play = false;
+            ballXdir = 0;
+            ballYdir = 0;
+            g.setColor(Color.green);
+            g.setFont(new Font("serif", Font.BOLD, 30));
+            g.drawString("You Won !!! Score:" + score, 190, 300);
+
+            g.setFont(new Font("serif", Font.BOLD, 20));
+            g.drawString("Press Enter to Restart", 230, 350);
+
+        }
+
+        if (ballposY > 570) {
+            play = false;
+            ballXdir = 0;
+            ballYdir = 0;
+            g.setColor(Color.red);
+            g.setFont(new Font("serif", Font.BOLD, 30));
+            g.drawString("Game Over :D Score:" + score, 190, 300);
+
+            g.setFont(new Font("serif", Font.BOLD, 20));
+            g.drawString("Press Enter to Restart", 230, 350);
+        }
+
+        g.dispose();
+
+
+    }
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        timer.start();
+        if (play) {
+            if (new Rectangle(ballposX, ballposY, 20, 20).intersects(new Rectangle(playerX, 550, 100, 5))) {
+                ballYdir = -ballYdir;
+            }
+            A:
+                for (int i = 0; i < map.getWall().length; i++) {
+                    for (int j = 0; j < map.getWall()[0].length; j++) {
+                        if (map.getWall()[i][j] > 0) {
```

Comment:
It would be better if `map.getWall()` was stored in a variable before the `for` loop and then using the variable here, instead of calling the getter thrice